### PR TITLE
Changes made to the Homepage

### PR DESCRIPTION
**Pull request by smridhiwho**
I made the following changes:

-  changed the background, and the navigation bar  
- added some text & animation to the landing page. 
- added an about section at the ended of the homepage 
- copied the nav-bar changes to the "Image to pdf" page as well 

Git page : https://smridhiwho.github.io/Student-portal/

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -102,30 +157,33 @@ main h2 {
...
+
 main .row {
   display: flex;
+  flex-wrap: nowrap;
   justify-content: center;
   align-items: center;
-  box-shadow: 0 4px 8px 0 #020225a9, 0 6px 20px 0 #020225a9;
+  box-shadow: 0 4px 8px 0 #78787ea9, 0 6px 20px 0 #3c3c3da9;
   padding: 10px 20px;
   margin: 20px 0;
```
Comment (thepranaygupta) [inline home.css]: change the values in px to rem

**Review by thepranaygupta — CHANGES_REQUESTED** [review]

**Comment by aakankshabhende** [comment]
@smridhiwho can you please put the git page link here

File: home.css (hunk 1 above)
Comment (smridhiwho) [reply home.css:179]: Sure

Comment (smridhiwho) [reply home.css]: Okay

**Comment by smridhiwho** [comment]
> @smridhiwho can you please put the git page link here

Done.

**Comment by aakankshabhende** [comment]
Great work :) @smridhiwho , just resolve the branch conflicts

**Review by thepranaygupta — APPROVED** [review]
LGTM 👍

**Review by Abirpal202049 — APPROVED** [review]

**Review by saikatmu31 — APPROVED** [review]

**Comment by aakankshabhende** [comment]
Please resolve the conflicts of this branch to get it merged in the main @smridhiwho

**Comment by Abirpal202049** [comment]
Make the changes after pulling the latest changes

**Pull request closed without merging**